src/Mod/Arch/ArchSite.py

```diff
@@ -278,14 +278,14 @@ def Activated(self):
                 else:
                     warning = True
         if warning :
-            message = translate( "Arch" ,  "Please select only Building objects or nothing!\n\
-Site are not allowed to accept other object than Building.\n\
-Other objects will be removed from the selection.\n\
-You can change that in the preferences.") + "\n"
+            message = translate( "Arch" ,  "Please either select only Building objects or nothing at all!\n\
+                                            Site is not allowed to accept any other object besides Building.\n\
+                                            Other objects will be removed from the selection.\n\
+                                            Note: You can change that in the preferences." )
```

Comment:
is this correct formatting ?

Reply:
No, it's not. It will add many spaces in between the text.